clang/test/Preprocessor/cheri-cap-sizes.c

```diff
@@ -1,3 +1,4 @@
+// REQUIRES: mips-registered-target
```

Comment:
Should not be needed for Driver tests. I wonder which line fails?

Reply:
Technically yes, but in this case clang forwards the `-mxcaptable` parameter to the backend, which is only supported by mips.
https://github.com/CTSRD-CHERI/llvm-project/blob/master/clang/lib/Driver/ToolChains/Clang.cpp#L1964
https://github.com/CTSRD-CHERI/llvm-project/blob/master/llvm/lib/Target/Mips/MipsISelLowering.cpp#L97

Reply:
That shouldn't matter...

Reply:
Ah and if that file is not built then the flag is not accepted. Should probably just modify the test to not actually consume the flag but just check the generated command line.